<launch>

        <node pkg="cvision" type="videoBridge.py"   name="videoBridge"  required="true" >
        </node>

        <node pkg="cvision" type="getWhiteBlob.py"  name="getWhiteBlob" required="true" >
        </node>
        
        <node pkg="cvision" type="getCircle.py"     name="getCircle"    required="true" >
        </node>
        
        <node pkg="cvision" type="getCorners.py"    name="getCorners"   required="true" >
        </node>
        
        <node pkg="cvision" type="getLaunchpad.py"  name="getLaunchpad" required="true" >
        </node>

</launch>
